acquire/acquire.py

```diff
@@ -1449,15 +1449,19 @@ def private_key_filter(path: fsutil.TargetPath) -> bool:
 @register_module("--home")
 class Home(Module):
     SPEC = [
+        # Catches most shell related configuration files
         ("glob", ".*[akz]sh*", from_user_home),
-        ("dir", ".config", from_user_home),
         ("glob", "*/.*[akz]sh*", from_user_home),
-        ("glob", "*/.config", from_user_home),
+        # Added to catch any shell related configuration file not caught with the above regex
         ("glob", ".*history", from_user_home),
         ("glob", "*/.*history", from_user_home),
         ("glob", ".*rc", from_user_home),
         ("glob", ".*_logout", from_user_home),
+        # Miscellaneous configuration files
+        ("dir", ".config", from_user_home),
+        ("glob", "*/.config", from_user_home),
         ("glob", "*/.wget-hsts", from_user_home),
+        ("glob", "*.gitconfig", from_user_home),
```

Comment:
```suggestion
        ("glob", ".gitconfig", from_user_home),
```
I think?

Reply:
No, it should be "file" instead of "glob", my bad